eq-author/src/App/Submission/Preview/SubmissionPreview/index.js

```diff
@@ -170,38 +157,6 @@ const SubmissionEditor = ({ submission, questionnaireTitle }) => {
           </Section>
```

Comment:
On the line before the closing Section tag at 157:
```
    {feedback && <SectionSeparator />}
```
This will conditionally render the prototype's separator below the view / print answers content when feedback is enabled.